README.md

```diff
@@ -1,20 +1,74 @@
 # Bonita Studio
 
-This repository contains the source code of the Bonita Studio component from Bonita Platform.  
-In order to build this repository it is required to build its dependencies. To help you achieve this there is a dedicated repository [here](https://github.com/Bonitasoft-Community/Build-Bonita).
+Development environment to build your applications with Bonita.
+//TODO screenshot studio global
```

Comment:
The Studio look might evolve significantly in next versions, adding a Screenshot here means that we'll have to maintain it, do we want it?